agent/agent_test.go

```diff
@@ -928,6 +1138,20 @@ func (s *AgentSuite) getPanicLogs() []string {
 	return panicLogs
 }
 
+// getIdleTimeoutLogs returns logs setting the idle timeout.
+func (s *AgentSuite) getIdleTimeoutLogs() []string {
+	var panicLogs []string
+	msgsByTask := s.mockCommunicator.GetMockMessages()
+	for _, msgs := range msgsByTask {
+		for _, msg := range msgs {
+			if strings.Contains(msg.Message, "Set idle timeout") {
+				panicLogs = append(panicLogs, msg.Message)
```

Comment:
This function is already going away, but I would avoid reusing this variable name because this function has nothing to do with panics.

Reply:
yeah I have since cleaned it up and then removed it